pkg/handlers/generic/mutation/imageregistries/credentials/inject_test.go

```diff
@@ -140,254 +142,271 @@ var _ = Describe("Generate Image registry patches", func() {
 		return mutation.NewMetaGeneratePatchesHandler("", client, NewPatch(client)).(mutation.GeneratePatches)
 	}
 
-	testDefs := []capitest.PatchTestDef{
+	testDefs := []struct {
+		capitest.PatchTestDef
+		assertOwnerReference bool
+	}{
 		{
-			Name: "unset variable",
+			PatchTestDef: capitest.PatchTestDef{
```

Comment:
No real change to the input here, just needed to indent existing tests under `capitest.PatchTestDef`